src/user-utilities.test.js

```diff
@@ -51,27 +51,7 @@ function testGetFullName(){
     }
 }
 
```

Comment:
If you would like to delete code, I would prefer you would comment it out rather than delete it and possibly talk to me offline. Jen's mad btw >:(